### PR TITLE
OPS-785: deployment script for firebase

### PR DESCRIPTION
**Pull request by adarshmishra**
### What do you want to achieve?
- deployment script for firebase. 
- NOTE: Project is currently selected by default. Ops team will create pipeline and pass project and firebase credentials dynamically. 

### PR Checklist

- [X] I have tested it locally and all functionalities are working fine.
- [ ] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

- https://widget-release.kommunicate.io/  Currently hosted on Firbase

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

### Screenshot
- 

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

**Review by devashishmamgain — CHANGES_REQUESTED** [review]
check for cache period setting https://github.com/Kommunicate-io/Kommunicate-Web-SDK/blob/master/customHttp.yml

File: firebase.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "headers": [
+      {
+        "source": "**/*.html",
+        "headers": [
+          {
+            "key": "Access-Control-Allow-Origin",
+            "value": "*"
+          }
+        ]
+      },
```
Comment (panwaranuj01) [inline firebase.json:18]: There is one file, `kommunicate.app`, which is the initial point of the script basically, we are giving that URL in the customer script, and it does not follow any file convention these regex will not match that file

**Review by panwaranuj01 — CHANGES_REQUESTED** [review]

**Review by panwaranuj01 — COMMENTED** [review]
And we also have to add this is header for only this file kommunicate.app
```
  - pattern: "/v2/kommunicate.app"
    headers:
      - key: "Content-Type"
        value: "text/javascript"
      - key: "Cache-Control"
        value: "public, max-age=180"

```

File: firebase.json (hunk 1 above)
Comment (adarshmishra) [reply firebase.json:18]: added

**Comment by adarshmishra** [comment]
> check for cache period setting https://github.com/Kommunicate-io/Kommunicate-Web-SDK/blob/master/customHttp.yml

Converted according to above file, @devashishmamgain @anujpanwar01

**Review by panwaranuj01 — APPROVED** [review]

**Pull request merged**